Cargo.toml

```diff
@@ -87,6 +89,12 @@ curve25519-dalek = { version = "3", optional = true, default-features = false, f
 x25519-dalek = { version = "1.1", optional = true, default-features = false, features = [ "u64_backend" ] }
 zeroize = { version = "1.3", optional = true, default-features = false }
 
+
+[dependencies.subtle]
```

Comment:
I'd be nice to keep all dependencies in the same style, eg. `subtle = { version = "2.4", optional = true, default-features = false }`